web-services/query/src/main/java/datawave/webservice/query/runner/BasicQueryBean.java

```diff
@@ -69,6 +69,7 @@
 public class BasicQueryBean {
     
     private static final String PRIVILEGED_USER = "PrivilegedUser";
+    private static final String UNLIMITED_QUERY_RESULTS_USER = "UnlimitedQueryResultsUser";
```

Comment:
FWIW, I don't think anyone uses this query wizard stuff.  It's not going to be carried forward to the query microservices.

Reply:
Okay, will remove